CONTRIBUTORS.md

```diff
@@ -15,6 +15,7 @@
 * Jason Gilman
 * Jenny Liu
 * Jonathan Kovarik
+* Juanisa McCoy
```

Comment:
! Sorry!!! I feel bad, as I reviewed the PR that added the contrib auto-generation.  😓     We should make sure that the generation script pulls your name properly from the github committers, or if it gets added to the automation it might pull it again.

Reply:
No worries.